### PR TITLE
Allow PMs to edit permitted project fields

### LINKED ISSUE #701: Project Detail Edit: Hide info that needs approval from PM view
### Overview
As a Org rep, I need to have pms be able to only change things themselves on their project object that do not require approval from Org so that we can reduce the amount of time spent on editing of database on their behalf

### Details
Additional options will be available to admin level and in future the pms will be able to change those items through a review process that will exist in 0.5. Also, we want them to be able to see all their info, with a note saying something about reach out to the #admin slack channel to change grayed out items.

### Action Items
Review list of items and put a check box by anything they should be able to change.

### Resources/Instructions
List of items that can be edited for bonnie to write a requirements doc for hiding

- [ ] Name: 
- [ ] Description: 
- [ ] GitHub URL: 
- [ ] Slack URL: 
- [x] Google Drive URL:
- [ ] HfLA Website URL: 
- [ ] Domain Name URL: 
- [x] Video Conference Link: 
- [x] Partners (comma separated): 
- [x] Times for the meetings
- [ ] Location: remote

### PR DESCRIPTION
**Pull request by chukalicious**
Closes #701

### Description:

This PR restricts the updating of specific fields in the project object, granting full access to editable fields to users with 'admin' access level, and restricted access to users with 'user' access level. 

### Action taken:

- Created a canEdit prop for the field components that's an array of types of users allowed to make changes to the field
- Compared the logged in user's access level to the values in the canEdit prop
- If the access level can be found in the array the edit button will be enabled, if not, a link that navigates to a readme doc with instructions will be rendered

### CODE REVIEW

File: client/src/components/manageProjects/editProject.js [hunk 1]
```diff
@@ -1,207 +1,237 @@
...
 import EditableField from './editableField';
-import { REACT_APP_CUSTOM_REQUEST_HEADER } from "../../utils/globalSettings";
+import { REACT_APP_CUSTOM_REQUEST_HEADER } from '../../utils/globalSettings';
 
+// Need to hold user state to check which type of user they are and conditionally render editing fields in this component
+//for user level block access to all except for the ones checked
 
-const EditProjectInfo  = ( props ) => {
-
+const EditProject = (props) => {
```
Comment (drubgrubby) [inline client/src/components/manageProjects/editProject.js:9]: Were you an English major?  I love that the name of this component bothered you enough to change it here, and in all of the other places that it appears.  Miriam Webster would be proud of you.

**Review by drubgrubby — COMMENTED** [review]
@chukalicious => This is exactly the required functionality and very nicely done.  

Additionally, you write very tidy code which is easy to read.  Much appreciated.  

I'm going to wait to merge this as I think Bonnie may want to change the message to something specific that tells them who to contact if they need to make a change to one of the fields that they don't have access to.  We'll ask her tomorrow night. 

Really nice work!

**Comment by drubgrubby** [comment]
Per Bonnie, we should create a readme/markdown file in vrms and link to it from the error message.  The message on the project edit page should read something like: "You don't have access to edit this field. Click here to find out who does".  Or something better with fewer words.

File: client/src/components/manageProjects/editProject.js (hunk 1 above)
Comment (chukalicious) [reply client/src/components/manageProjects/editProject.js:9]: 😆 I can barely put two words of English together! But yeah, I find it confusing when the component and the file don't have the same name.

**Comment by chukalicious** [comment]
@drubgrubby Hi, I have created the readme file and linked it to the disabled edit button but maybe the link will need to be updated when the branch gets merged, unless you know of another way in which I can link the file without requesting a second merge. I hope that makes sense 😬

**Review by drubgrubby — APPROVED** [review]
In discussion with @chukalicious we have decide to merge this as is - which is working perfectly, but needing some changes to the links - and create a new issue to make those changes. 

That issue is #796

**Pull request merged**